pkg/controller/composite/controller.go

```diff
@@ -296,9 +296,15 @@ func (pc *parentController) processNextWorkItem() bool {
 	defer pc.queue.Done(key)
 
 	if err := pc.sync(key.(string)); err != nil {
-		utilruntime.HandleError(fmt.Errorf("failed to sync %v '%v': %w", pc.parentResource.Kind, key, err))
-		pc.queue.AddRateLimited(key)
-		return true
+		if apierrors.IsNotFound(err) {
```

Comment:
I think this is handled in https://github.com/metacontroller/metacontroller/blob/master/pkg/controller/composite/controller.go#L487-L490

Reply:
Thanks I'll poke around some more to see how I am entering that state